src/slim-select/render.ts

```diff
@@ -481,6 +490,18 @@ export default class Render {
         }
       }
     }
+
+    
+
+    // If the number of selected items goes above the threshold, hide the chips
```

Comment:
this would move up to line 445, after the part it check for selected length for placeholder. We essentially want to catch it before it does logic for rendering out each individual value.